src/questions/author-linkedin.spec.js

```diff
@@ -0,0 +1,14 @@
+const askAuthorLinkedIn = require('./author-linkedin')
+
+describe('askAuthorLinkedIn', () => {
+  it('should return correct question format', () => {
+    const result = askAuthorLinkedIn()
+
+    expect(result).toEqual({
+      type: 'input',
+      message: '🕴  LinkedIn username (use empty value to skip)',
```

Comment:
```suggestion
      message: '💼  LinkedIn username (use empty value to skip)',
```

Sorry forgot to suggest this one too

Reply:
That's why we have tests right? 😉